src/pocompiler.js

```diff
@@ -16,63 +19,69 @@ export default function (table, options) {
 }
 
 /**
- * Creates a PO compiler object.
+ * Takes the header object and converts all headers into the lowercase format
  *
- * @constructor
- * @param {Object} table Translation table to be compiled
+ * @param {{ [x: string]: any; }} headersRaw the headers to prepare
+ * @returns {{ [x: string]: any; }} the headers in the lowercase format
```

Comment:
I would use `Record` here too.

```suggestion
 * @param {Record<string, any>} headersRaw the headers to prepare
 * @returns {Record<string, any>} the headers in the lowercase format
```